--- isle-imageservices/rootfs/etc/confd/templates/imageserv/cantaloupe.properties.tpl	2021-04-23 15:49:30.000000000 -0400
+++ rootfs/etc/confd/templates/imageserv/cantaloupe.properties.tpl	2021-04-23 15:52:10.000000000 -0400
@@ -39,7 +39,7 @@
 # Base URI to use for internal links, such as Link headers and JSON-LD
 # @id values, in a reverse-proxy context. This should only be used when
 # X-Forwarded-* headers cannot be used instead. (See the user manual.)
-base_uri =
+base_uri = {{ getv "/base/uri" }}
 
 # Normally, slashes in a URI path component must be percent-encoded as
 # "%2F". If your proxy is not able to pass these through without decoding,
@@ -171,7 +171,7 @@
 
 # Read data in chunks when it may be more efficient. (This also may end up
 # being less efficient, depending on many variables; see the user manual.)
-HttpSource.chunking.enabled = true
+HttpSource.chunking.enabled = false
 
 # Chunk size.
 HttpSource.chunking.chunk_size = 512K
@@ -340,7 +340,7 @@
 # dealt with.
 # * `DownloadStrategy` and `CacheStrategy` work the same as above.
 # * `AbortStrategy` causes the request to fail.
-processor.fallback_retrieval_strategy = DownloadStrategy
+processor.fallback_retrieval_strategy = CacheStrategy
 
 # Resolution of vector rasterization (of e.g. PDFs) at a scale of 1.
 processor.dpi = 150
@@ -465,20 +465,20 @@
 # for forever.
 # Per https://github.com/Islandora-Collaboration-Group/ISLE/issues/207
 # Changed from default 259200 (30 days) to 86400 (1 day)
-cache.server.source.ttl_seconds = 86400
+cache.server.source.ttl_seconds = 604800
 
 # Enables the derivative (processed image) cache.
 cache.server.derivative.enabled = true
 
 # Available values are `FilesystemCache`, `JdbcCache`, `RedisCache`,
 # `HeapCache`, `S3Cache`, and `AzureStorageCache`.
-cache.server.derivative = FilesystemCache
+cache.server.derivative = {{ getv "/cantaloupe/derivative/strategy" }}
 
 # Amount of time derivative cache content remains valid. Set to blank or 0
 # for forever.
 # Per https://github.com/Islandora-Collaboration-Group/ISLE/issues/207
 # Changed from default 259200 (30 days) to 86400 (1 day)
-cache.server.source.ttl_seconds = 86400
+cache.server.derivative.ttl_seconds = {{ getv "/cantaloupe/derivative/ttl/seconds" }}
 
 # Whether to use the Java heap as a "level 1" cache for image infos, either
 # independently or in front of a "level 2" derivative cache (if enabled).
@@ -567,20 +567,20 @@
 # !! Endpoint URI.
 # For AWS endpoints, see:
 # https://docs.aws.amazon.com/general/latest/gr/rande.html#s3_region
-S3Cache.endpoint =
+S3Cache.endpoint = {{ getv "/cantaloupe/s3cache/endpoint" }}
 
 # !! Credentials for your AWS account.
 # See: http://aws.amazon.com/security-credentials
 # Note that this info can be obtained from elsewhere rather than setting it
 # here; see the user manual.
-S3Cache.access_key_id =
-S3Cache.secret_key =
+S3Cache.access_key_id = {{ getv "/cantaloupe/s3cache/access/key/id" }}
+S3Cache.secret_key = {{ getv "/cantaloupe/s3cache/secret/key" }}
 
 # !! Name of a bucket to use to hold cached data.
-S3Cache.bucket.name =
+S3Cache.bucket.name = {{ getv "/cantaloupe/s3cache/bucket/name" }}
 
 # !! String that will be prefixed to object keys.
-S3Cache.object_key_prefix =
+S3Cache.object_key_prefix = {{ getv "/cantaloupe/s3cache/object/key/prefix" }}
 
 # !! Maximum number of concurrent HTTP connections to AWS. Leave blank to
 # use the default.
